server/src/Ogmios/Query.hs

```diff
@@ -89,7 +89,7 @@ tryQueryUntilZero query remainAttempts
       Left e -> do
         putStrLn $ "Error : " <> show e
         putStrLn "Waiting for ogmios conection attempt"
-        putStrLn $ "Attempts remain : " <> show (remainAttempts -1)
+        putStrLn $ "Attempts remain : " <> show (remainAttempts - 1)
```

Comment:
This just made me realize we don't need any of this module anymore (we removed a lot of server code recently). This is more of a note to myself to remember to remove it